go.mod

```diff
@@ -5,6 +5,7 @@ go 1.16
 require (
 	github.com/BurntSushi/toml v1.0.0
 	github.com/alessio/shellescape v1.4.1
+	github.com/container-orchestrated-devices/container-device-interface v0.5.5-0.20230516140309-1e6752771dc5 // indirect
```

Comment:
This unfortunately seems to pull in many dependencies, one of our core tenants is being extremely easy to embed and we also have to ensure we're in compliance with kubernetes / CNCF dependency licensing policies.

Maybe we should just leave validation to the underlying container runtimes?

Reply:
Yes, I think underlying validation should be fine. I'll clean it up.